exercises/B-hello-world/exercise.js

```diff
@@ -1 +1,3 @@
 console.log("Hello world");
+console.log("I am practicing JS.");
+console.log("It's \"Nov 2022\".");
```

Comment:
Nice, you can either escape the " like you have done here, or wrap the whole string in single quotes `'`